CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+[42.5.0] (2022-08-23 11:20:11 -0400)
```

Comment:
Versions should start as a header with the prefix ##

`## [42.5.0] (2022-08-23 11:20:11 -0400)`

(42.4.1 and 42.4.2 have the same issue)

Reply:
good catch, thx